test/templates/nstemplatetiers/appstudio/ns_tenant.yaml

```diff
@@ -13,6 +13,7 @@ objects:
     labels:
       name: ${USERNAME}-tenant
       toolchain.dev.openshift.com/workspace: ${USERNAME}
+      appstudio.redhat.com/workspace_name: ${USERNAME}
```

Comment:
there is no need to update also the test template - this is just a dummy thing.

Reply:
ack

Reply:
ok, let me remove this. we dont need the billing labels everywhere.